assets/js/Containers/FrevaGPT/index.js

```diff
@@ -74,7 +81,11 @@ class FrevaGPT extends React.Component {
       this.setState({ botModelList: await response.json()});
     }
 
-    getBotModels();
+    if (pingBot() !== "200") {
```

Comment:
`response.status` is a Number, not a String. Change `"200"` to `200`, otherwise the bot will always be unavailable.

Furthermore, you have to await `pingBot`, otherwise it will also not work because you don't get any response on time. This also means that `componentDidMounts` has to be `async`

Reply:
Note resolved: https://github.com/FREVA-CLINT/freva-web/pull/61/commits/27e1b060aff9542b11120a97ce9a4a57c294671b